src/Router.js

```diff
@@ -1,17 +1,102 @@
-import React, { lazy, Suspense } from 'react';
+import React, { Suspense, lazy, useState } from 'react';
 
+import { Button } from '@patternfly/react-core';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
 import { useFlag } from '@unleash/proxy-client-react';
+import PropTypes from 'prop-types';
 import { Route, Routes } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 
 import EdgeImageDetail from './Components/edge/ImageDetails';
 import ShareImageModal from './Components/ShareImageModal/ShareImageModal';
+import { useGetComposesQuery } from './store/imageBuilderApi';
 import { manageEdgeImagesUrlName } from './Utilities/edge';
+import { resolveRelPath } from './Utilities/path';
 
 const LandingPage = lazy(() => import('./Components/LandingPage/LandingPage'));
 const CreateImageWizard = lazy(() =>
   import('./Components/CreateImageWizard/CreateImageWizard')
 );
 
+/**
+ * Help page for newcomers creating their first image if they've never did it.
+ * The user gets the possibility to click on a `create image` button that will
+ * open the wizard for them.
+ *
+ * caveat:
+ * This component takes care of displaying itself the CreateImageWizard. Meaning
+ * that the wizard is not being displayed using the navigation system. This is
+ * valid for-now™ [2023-09] because the wizard is being displayed in a modal.
+ * This would most likely have to get updated when the wizard will get changed
+ * to an InPage mode.
+ */
+const AppZeroState = () => {
+  const [wizardOpenned, setWizardOpenned] = useState(false);
```

Comment:
It isn't an official convention in our code base yet but I usually like to start boolean variable names with `is`, what do you think of `isWizardOpened`?

Reply:
Very good idea.